src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeExperiment.groovy

```diff
@@ -43,7 +43,9 @@ final class OxfordNanoporeExperiment implements ExperimentFolder {
             FQDN_FOLDERS + ".Fast5PassFolder",
             FQDN_FOLDERS + ".Fast5FailFolder",
             FQDN_FOLDERS + ".FastQPassFolder",
-            FQDN_FOLDERS + ".FastQFailFolder"
+            FQDN_FOLDERS + ".FastQFailFolder",
+            FQDN_FOLDERS + ".UnclassifiedFast5Folder",
+            FQDN_FOLDERS + ".UnclassifiedFastQFolder"
```

Comment:
```suggestion
            FQDN_FOLDERS + ".UnassignedFast5Folder",
            FQDN_FOLDERS + ".UnassignedFastQFolder"
```

Reply:
according to the schema, the naming convention is `unclassified`

Reply:
I was not up to date.